<launch>
  <node pkg="opencv_test" name="opencv_test" type="opencv_test" output="screen" >
    <!--    -->
    <param name="delaytime1"     type="double"     value="0.5  "/>  <!-- 无用  -->
    <param name="delaytime2"     type="double"     value="0.5  "/>  <!-- 无用  -->
    <param name="pictime"     type="double"     value="0.2  "/>     <!-- 每次拍照的停顿时间  -->
    <param name="vel"     type="double"     value="1.8  "/>         <!-- 每次转向的角速度增量Kp  -->
    <param name="debug"     type="int"     value="1 "/>            <!-- 是否自己设置转向角度，1不自己设置，0自己设置  -->
    <!-- 当debug设置为0时才生效，angle设置为100.0表示无效 -->
    <param name="angle1"     type="double"     value="100.0  "/>          <!-- 转向角度1,bu yong  -->
    <param name="angle2"     type="double"     value="0.46  "/>          <!-- 转向角度2  -->
    <param name="angle3"     type="double"     value="100.0  "/>          <!-- 转向角度3  -->
    
  </node>
</launch> 


